components/camel-casper/src/test/java/org/apache/camel/component/casper/producer/CasperProducerWith_STATE_ITEM_OperationTest.java

```diff
@@ -0,0 +1,88 @@
+package org.apache.camel.component.casper.producer;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.apache.camel.CamelExchangeException;
+import org.apache.camel.Exchange;
+import org.apache.camel.Produce;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.casper.CasperConstants;
+import org.apache.camel.component.casper.CasperTestSupport;
+import org.apache.commons.cli.MissingArgumentException;
+import org.junit.jupiter.api.Test;
+
+import com.syntifi.casper.sdk.model.storedvalue.StoredValue;
+
+@SuppressWarnings("rawtypes")
```

Comment:
Avoid suppressing warnings. Ideally, all warnings should be fixed prior to inclusion.

Reply:
fixed